mvc/src/main/java/com/interface21/webmvc/servlet/view/JsonView.java

```diff
@@ -1,14 +1,25 @@
 package com.interface21.webmvc.servlet.view;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.interface21.web.http.MediaType;
 import com.interface21.webmvc.servlet.View;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
-
 import java.util.Map;
 
 public class JsonView implements View {
 
     @Override
-    public void render(final Map<String, ?> model, final HttpServletRequest request, HttpServletResponse response) throws Exception {
+    public void render(final Map<String, ?> model, final HttpServletRequest request, HttpServletResponse response)
+            throws Exception {
+        ObjectMapper objectMapper = new ObjectMapper();
+        if (model.size() == 1) {
+            Object value = model.values().iterator().next();
+            objectMapper.writeValue(response.getWriter(), value);
+            return;
+        }
+        String body = objectMapper.writeValueAsString(model);
```

Comment:
개수에 따른 분기를 한 이유가 있을까요?
```suggestion
        objectMapper.writeValue(response.getWriter(), model);
```

Reply:
오 이 부분은 제가 힌트 아래를 안 봐서 몰랐네요. 반영하지 않아도 됩니다 ✅

Reply:
> model에 데이터가 1개면 값을 그대로 반환하고 2개 이상이면 Map 형태 그대로 JSON으로 변환해서 반환한다.

다시 보니 살짝 애매하긴 하네요. 일반적인 `Json`을 생각한다면 value만 뽑아내는 것이 어색하기도 하고요. **값을 그대로 반환**한다는 건 Model의 Key-Value를 하나의 값으로 보았을까요? 🤔 저는 그렇게 생각해서 전부 담긴 했습니다. Jackson이 알아서 해 주니깐요..

Reply:
Model의 key-value를 모두 담기 시작하는 순간 아루가 제안주신 것처럼 모델 사이즈에 따른 분기가 필요없겠네요.

저도 해당 요구사항의 의도를 파악하진 못했는데요.. 우선은 값을 Model map의 키값이 없는 값만으로 해석하기는 했습니다~
나중에 구구에게 의도를 물어봐야겠군요 🤔